flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/stream/sql/MatchRecognizeITCase.scala

```diff
@@ -311,6 +312,61 @@ class MatchRecognizeITCase extends StreamingWithStateTestBase {
     assertEquals(expected.sorted, StreamITCase.testResults.sorted)
   }
 
+  @Test
+  def testWindowedGroupingAppliedToMatchRecognize(): Unit = {
+    val env = StreamExecutionEnvironment.getExecutionEnvironment
+    env.setParallelism(1)
+    env.setStreamTimeCharacteristic(TimeCharacteristic.EventTime)
+    val tEnv = TableEnvironment.getTableEnvironment(env)
+    StreamITCase.clear
+
+    val data = new mutable.MutableList[(String, Long, Int, Int)]
+    //first window
+    data.+=(("ACME", Time.seconds(1).toMilliseconds, 1, 1))
+    data.+=(("ACME", Time.seconds(2).toMilliseconds, 2, 2))
+    //second window
+    data.+=(("ACME", Time.seconds(4).toMilliseconds, 1, 4))
+    data.+=(("ACME", Time.seconds(5).toMilliseconds, 1, 3))
+
+
+    val t = env.fromCollection(data)
```

Comment:
please, do not use one letter variables. Can you name `t` based on it's content? Like `sales`? `orders`? `prices`?